Project21/Project21/Testcase.c

```diff
@@ -0,0 +1,74 @@
+// ------------------------------------------------------------------------------------------------
+// Assignment 3 ~ A training program for new joiners at Metamation, Batch - July 2024.
+// Copyright (c) Metamation India.Naveenkumar s
+// ------------------------------------------------------------------
+// Testcase.c
+// Decimal to Binary conversion and Decimal to HexaDecimal conversion
+// ------------------------------------------------------------------------------------------------
+#include"conversionfile.h"
+
+void PrintResult (int input[],char* output[],char*(*func)(int)) {
+   for (int i = 0; i < 11; i++) {

```

Comment:
Please do not hard code the values. Store the count of the test cases and use it.